flake-modules/dev/list-plugins/list-plugins.py

```diff
@@ -252,6 +256,12 @@ def main(args) -> None:
     """,
         formatter_class=RawTextHelpFormatter,
     )
+    parser.add_argument(
+        "--root-path",
+        type=str,
+        default="./",
```

Comment:
Maybe as a TODO comment:

This could default to looking recursively for the flake.lock like some of the other scripts do?

Reply:
Maybe we want to have some lib/util for common logic used in most/all our scripts?

Reply:
Well, overkilled for this PR, but why not.
I thought about setting the `devShell` command to `--root-path ${self}`, but then it would not have been dynamic anymore.

Reply:
Good point. Normally we can't use `self` because that's the in-store flake and we often want the flake in the local git repo instead.

I'm guessing here it would be ok to use the in-store flake (`self`)?